src/integrationtest/native/cpp/SimulationTest.cpp

```diff
@@ -119,8 +119,8 @@ class IntegrationTest : public ::testing::Test {
       if (m_settings.mechanism == sysid::analysis::kArm) {
         // The acceleration fit (used only for arms) is worse than the next
         // velocity fit
-        EXPECT_NEAR(Kv, ffGains[1], 0.015);
-        EXPECT_NEAR(Ka, ffGains[2], 0.04);
+        EXPECT_NEAR(Kv, ffGains[1], 0.01);
+        EXPECT_NEAR(Ka, ffGains[2], 0.01);
```

Comment:
How many tests have you done with these values?

Reply:
Two, including this one.

Reply:
Looks like we need to increase kA?

Reply:
It's off by like 0.07. :/

Reply:
Mac is cursed